test/functional/feature_logging.py

```diff
@@ -67,7 +67,7 @@ def run_test(self):
 
         # just sanity check no crash here
         self.stop_node(0)
-        self.start_node(0, ["-debuglogfile=/dev/null"])
+        self.start_node(0, ["-debuglogfile=%s" % os.devnull])
```

Comment:
In commit "stop printing to console in test runner"

This change looks like it should go in the previous commit ("Enable -printtoconsole by default if bitcoind is not a daemon").